src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -115,4 +116,10 @@ public void parseCommand_unknownCommand_throwsParseException() {
     public void trackCommand_track() throws Exception {
         assertTrue(parser.parseCommand(TrackCommand.COMMAND_WORD + " student") instanceof TrackCommand);
     }
+
+    @Test
+    public void parseCommand_deletetag() throws Exception {
+        assertTrue(parser.parseCommand(
+                DeleteTagCommand.COMMAND_WORD + " 1" + " t/test") instanceof DeleteTagCommand);
```

Comment:
Perhaps consider changing to use constants to maintain consistency. 
```suggestion
                DeleteTagCommand.COMMAND_WORD + INDEX_FIRST_PERSON.getOneBased() + " t/test") instanceof DeleteTagCommand);
```